### ISSUE TITLE
Add docker images for armv7 and arm64

### ISSUE THREAD

**Issue by patrickelectric** [xiaojun207/docker-manager#35]
Right now only amd64 exists, it would be nice to have armv7 and arm64 docker images.

**Comment by xiaojun207** [comment]
Yes, I currently do not have an ARM testing environment here. If you have an ARM environment, you can try creating an ARM image and let me know.
